GUIDE.md

```diff
@@ -0,0 +1,181 @@
+# Node Documentation
+
+Welcome to the Node.js documentation guide – or more colloquially, the docs on
+docs. If you're here, you're likely looking for guidance on how to write
```

Comment:
I like the engaging intro. There are some copyediting nits:

* That looks like an en dash. It should be an em dash. (You can use HTML entities in markdown, so I'd recommend using the mdash entity here.)

* No spaces before or after the dash.

* "Node.js Documentation Guide" rather than "Node.js documentation guide"

* If you think this will be read by many non-native speakers, you may be doing them a favor by avoiding words like *colloquially*.